Lib/multiprocessing/semaphore_tracker.py

```diff
@@ -43,10 +45,16 @@ def ensure_running(self):
         with self._lock:
             if self._pid is not None:
                 # semaphore tracker was launched before, is it still running?
-                pid, status = os.waitpid(self._pid, os.WNOHANG)
-                if not pid:
-                    # => still alive
-                    return
+                try:
+                    pid, _ = os.waitpid(self._pid, os.WNOHANG)
+                except ChildProcessError:
+                    # The process must be death (no process with pid self._pid).
```

Comment:
I would just say something like "# The process terminated". No need to elaborate.

Reply:
Agreed with Victor.